Simplenote/Classes/SPNoteTableViewCell.swift

```diff
@@ -226,6 +246,21 @@ private extension SPNoteTableViewCell {
         bodyLabel.numberOfLines = Style.maximumNumberOfBodyLines
         bodyLabel.lineBreakMode = .byWordWrapping
     }
+
+    private func setupCellSeparatorView() {
+        separatorsView = SeparatorsView()
```

Comment:
You can initialize this one directly at the property level, and you can get away without the `var` and the IUO

```
private let separatorsView = SeparatorsView()
```